pandas/tests/io/test_sql.py

```diff
@@ -552,6 +680,57 @@ def postgresql_psycopg2_conn(postgresql_psycopg2_engine):
         yield conn
 
 
+@pytest.fixture
+def postgresql_adbc_conn():
+    pytest.importorskip("adbc_driver_postgresql")
+    from adbc_driver_postgresql import dbapi
+
+    uri = "postgresql://postgres:postgres@localhost:5432/pandas"
+    with dbapi.connect(uri) as conn:
+        yield conn
+        for view in get_all_views(conn):
+            drop_view(view, conn)
+        for tbl in get_all_tables(conn):
+            drop_table(tbl, conn)
+        conn.commit()
+
+
+@pytest.fixture
+def postgresql_adbc_iris(postgresql_adbc_conn, iris_path):
+    import adbc_driver_manager as mgr
+
+    conn = postgresql_adbc_conn
+
+    try:
+        conn.adbc_get_table_schema("iris")
+    except mgr.ProgrammingError:
```

Comment:
Just for my understanding, this is adbc's way of "if iris doesn't exist, then load the data"?